scripts/docker/build-binaries.bash

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+. "/ci/shared/helpers/git-helpers.bash"
+
+pushd /repo > /dev/null
+git_configure_safe_directory
+REPO_NAME=$(git_get_remote_name)
+popd > /dev/null
+
+. <(/ci/shared/helpers/extract-default-params-for-task.bash /ci/shared/tasks/build-binaries/linux.yml)
+
+export DEFAULT_PARAMS="/ci/$REPO_NAME/default-params/build-binaries/linux.yml"
+pushd / > /dev/null
+/ci/shared/tasks/build-binaries/task.bash
```

Comment:
@winkingturtle-vmw Hi! I'm running this task in a docker container, but the build-binaries task is getting an unbound variable error on `$MAPPING`. 
```
root@55a995012ac3:/# repo/scripts/docker/build-binaries.bash
/ci/shared/tasks/build-binaries/task.bash: line 22: MAPPING: unbound variable
```
Maybe it should get set here?

Reply:
Nevermind, I pulled the `wg-app-platform-runtime-ci` repo and it passes!